src/Generator/TypeGenerator.php

```diff
@@ -33,7 +33,8 @@
      *
      * @link http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration
      */
-    private static $internalPhpTypes = ['void', 'int', 'float', 'string', 'bool', 'array', 'callable', 'iterable'];
+    private static $internalPhpTypes = ['void', 'int', 'float', 'string', 'bool', 'array', 'callable', 'iterable',
+        'object'];
```

Comment:
What's the right alignment here ? I try to find a "too" long array property but can't find one in the project...

Is this ok?

```
private static $internalPhpTypes = [
    'void', 
    'int', 
    'float', 
    'string', 
    'bool', 
    'array', 
    'callable', 
    'iterable',
    'object'
];
```